dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilder.idl

```diff
@@ -31,6 +31,14 @@ namespace Microsoft.Windows.AppNotifications.Builder
         Critical,
     };
 
+    [contract(AppNotificationBuilderContract, 2)]
+    enum AppNotificationButtonSettingType
```

Comment:
We prefer to not use the word "type" in type names. :) Maybe this is `AppNotificationButtonStyle` ?

Reply:
AppNotificationButtonStyle already exists, will rename to AppNotificationButtonSettingStyle